conf/logback-test.xml

```diff
@@ -8,11 +8,13 @@
         </encoder>
     </appender>
 
-    <logger name="application" level="${logger.application:-DEBUG}">
+    <logger name="application" level="${logger.application:-WARN}">
         <appender-ref ref="STDOUT"/>
     </logger>
 
-    <root level="${logger.application:-WARN}">
+    <logger name="pekko" level="ERROR"/>
+
+    <root level="${logger.application:-ERROR}">
```

Comment:
Should root stay on `WARN`? I realise it's for tests only, so checking in case it was accidental.

Reply:
Noisy tests